CMakeLists.txt

```diff
@@ -0,0 +1,130 @@
+# Set minimum CMake version to latest version on Gadi:
+cmake_minimum_required(VERSION 3.24.2)
+
+project(
+    CABLE
+    LANGUAGES Fortran
+)
+
+option(CABLE_MPI "Build the MPI executable" OFF)
+
+if(CABLE_MPI)
+    # The MPI implementation is currently not supported in CABLE-POP_TRENDY:
+    message(FATAL_ERROR "MPI compilation not supported.")
+endif()
+
+# third party libs
+if(CABLE_MPI)
+    find_package(MPI REQUIRED COMPONENTS Fortran)
+endif()
+find_package(PkgConfig REQUIRED)
+pkg_check_modules(NETCDF REQUIRED IMPORTED_TARGET "netcdf-fortran")
+
+set(CABLE_Intel_Fortran_FLAGS -fp-model precise -fpp -nofixed -assume byterecl -D__INTEL__ -D__INTEL_COMPILER__ -D__NETCDF3__)
+set(CABLE_Intel_Fortran_FLAGS_DEBUG -O0 -g -traceback -fpe0 -debug extended -check all -warn all -fp-stack-check)
+set(CABLE_Intel_Fortran_FLAGS_RELEASE -O3 -ip)
+if(CMAKE_Fortran_COMPILER_ID MATCHES "Intel")
+    add_compile_options(
+        ${CABLE_Intel_Fortran_FLAGS}
+        "$<$<CONFIG:Release>:${CABLE_Intel_Fortran_FLAGS_RELEASE}>"
+        "$<$<CONFIG:Debug>:${CABLE_Intel_Fortran_FLAGS_DEBUG}>"
+    )
+endif()
+
+set(CABLE_GNU_Fortran_FLAGS -cpp -ffree-form -ffixed-line-length-132 -D__GFORTRAN__ -D__gFortran__ -D__CRU2017__ -D__NETCDF3__ -D__C13DEBUG__)
+set(CABLE_GNU_Fortran_FLAGS_DEBUG -O -g -pedantic-errors -Wall -W -Wno-maybe-uninitialized -fbacktrace -ffpe-trap=zero,overflow,underflow -finit-real=nan)
+set(CABLE_GNU_Fortran_FLAGS_RELEASE -O3 -Wno-aggressive-loop-optimizations)
```

Comment:
One can remove `-D__CRU2017__`, which is old and probably not used anymore and `-D__C13DEBUG__`, which is only for me when working on 13C.